TileFormats/VectorData/README.md

```diff
@@ -1,19 +1,248 @@
+
+If `featureTableJSONByteLength` equals zero, the tile does not need to be rendered.
+
+The body section immediately follows the header section, and is composed, in order, of four fields: `Feature Table`, `Batch Table`, `Indices`, `Positions`, and `Meshes`.
+
+Code for reading the header can be found in
+[Vector3DModelTileContent.js](https://github.com/AnalyticalGraphicsInc/cesium/blob/vector-tiles/Source/Scene/Vector3DTileContent.js)
+in the Cesium implementation of 3D Tiles.
+
+## Feature Table
+
+Contains values for `vctr` semantics used to render features.  The general layout of a Feature Table is described in the [Feature Table specification](../FeatureTable).
+
+The `vctr` Feature Table JSON schema is defined in [vctr.featureTable.schema.json](../../schema/vctr.featureTable.schema.json).
+
+### Semantics
+
+Per-feature semantics specific to a feature type are prefixed with the name of the feature type. e.g. `POLYGON` for pre-triangulated polygons, `POLYLINE` for polylines and `POINT` for points.
+
+At least one global `LENGTH` semantic must be defined. 
+If `POLYGONS_LENGTH` is not defined, or zero, no polygons will be rendered. 
+If `POLYLINES_LENGTH` is not defined, or zero, no polylines will be rendered.
+If `POINTS_LENGTH` is not defined, or zero, no points will be rendered.
+If `MESHES_LENGTH` in not defined, or zero, no meshes will be rendered.
+If `BOXES_LENGTH` in not defined, or zero, no boxes will be rendered.
+If `CYLINDERS_LENGTH` in not defined, or zero, no cylinders will be rendered.
+If `ELLIPSOIDS_LENGTH` in not defined, or zero, no ellipsoids will be rendered.
+If `SPHERES_LENGTH` in not defined, or zero, no spheres will be rendered.
+Multiple feature types may be defined in a single Vector tile using multiple `LENGTH` semantics, and in that case, all specified feature types will be rendered.
+
+If a semantic has a dependency on another semantic, that semantic must be defined as well.
+
+#### Global Semantics
+
+The semantics define global properties for all vector elements.
+
+| Semantic | Data Type | Description | Required |
+| --- | --- | --- | --- |
+| `POLYGONS_LENGTH` | `uint32` | The number of pre-triangulated polygons in the tile. | :red_circle: No. |
+| `POLYLINES_LENGTH` | `uint32` | The number of polylines in the tile. | :red_circle: No. |
+| `POINTS_LENGTH` | `uint32` | The number of points in the tile. | :red_circle: No. |
+| `MESHES_LENGTH` | `uint32` | The number of meshes in the tile. | :red_circle: No. |
+| `BOXES_LENGTH` | `uint32` | The number of boxes in the tile. | :red_circle: No. |
+| `CYLINDERS_LENGTH` | `uint32` | The number of cylinders in the tile. | :red_circle: No. |
+| `ELLIPSOIDS_LENGTH` | `uint32` | The number of ellipsoids in the tile. | :red_circle: No. |
+| `SPHERES_LENGTH` | `uint32` | The number of spheres in the tile. | :red_circle: No. |
+| `MINIMUM_HEIGHT` | `float32` | The minimum height for this tiles' region in meters above the WGS84 ellipsoid. | :white_check_mark: Yes. |
```

Comment:
Should min and max height be required?  These are just for potential rendering optimizations, right?  I think it is fine to make them optional.

Perhaps that will also play better if/when this tile is used for local coordinate systems.

Reply:
Should here and below have a `_ABOVE_WGS84` suffix?  They would not in Cesium, but in 3D Tiles, we don't really use "height."

Reply:
The min and max heights are required over the entire tile because they are used to interpolate the height values for points and polylines.

Is the suffix needed if it is defined to be above the ellipsoid in the description?

Reply:
Maybe not.  How can we generalize this so it is not specific to WGS84, e.g., if we just wanted to use it for a BIM model?  Should there be a tile-level enum that says WGS84 or a local coordinate?  Seems like this is becoming a common enough case.